examples/validators/gatekeeper-validate/invalid-configmap/.expected/results.yaml

```diff
@@ -0,0 +1,14 @@
+items:
```

Comment:
Can you add a TODO to test expected CLI output? (My previous comment)

This is important part of the UX.

Reply:
Adding it to the `.expected` directory messes up with the test.
I filed https://github.com/GoogleContainerTools/kpt/issues/1689 and added in m2 milestone.